CompetitionTemplate/src/competition/subsystems/turret/rotation/commands/RotateTurretWithGamepadCommand.java

```diff
@@ -25,6 +29,13 @@ public void initialize() {
 
     @Override
     public void execute() {
-        rotationSubsystem.setRotationPower(oi.leftJoystick.getButton(8) ? oi.leftJoystick.getRawAxis(4) : 0);
+        double leftIntent = leftTrigger.getVector().y;
+        double rightIntent = -1 * rightTrigger.getVector().y;
+        
+        if(rightIntent != 0 && leftIntent != 0){
```

Comment:
Never compare doubles for equality! Use a configurable threshold. `MathUtils` has some things for this, I believe (not completely sure).